pkg/common/scheduler/README.md

```diff
@@ -0,0 +1,256 @@
+
+3. **EvenPodSpread**: check if resources are evenly spread across pods [CORE]. It has an argument `MaxSkew` to configure the plugin with an allowed skew factor.
+
+### Priorities:
+
+1. **AvailabilityNodePriority**: make sure resources are evenly spread across nodes [CORE]. It has an argument `MaxSkew` to configure the plugin with an allowed skew factor.
+
+2. **AvailabilityZonePriority**: make sure resources are evenly spread across zones [CORE]. It has an argument `MaxSkew` to configure the plugin with an allowed skew factor.
+
+3. **LowestOrdinalPriority**: make sure vreplicas are placed on free smaller ordinal pods to minimize resource usage [CORE]
+
+**Example ConfigMap for config-scheduler:**
+
+```
+data:
+  predicates: |+
+                  [
+                    {"Name": "PodFitsResources"},
+                    {"Name": "NoMaxResourceCount",
+                    "Args": "{\"NumPartitions\": 100}"},
+                    {"Name": "EvenPodSpread",
+                    "Args": "{\"MaxSkew\": 2}"}
+                  ]
+  priorities: |+
+                  [
+                    {"Name": "AvailabilityZonePriority",
+                    "Weight": 10,
+                    "Args":  "{\"MaxSkew\": 2}"},
+                    {"Name": "LowestOrdinalPriority",
+                    "Weight": 2}
+                  ]
+```
+
+## Descheduler Profile:
+
+### Priorities:
+
+1. **RemoveWithAvailabilityNodePriority**: make sure resources are evenly spread across nodes [CORE]
+
+2. **RemoveWithAvailabilityZonePriority**: make sure resources are evenly spread across zones [CORE]
+
+3. **HighestOrdinalPriority**: make sure vreps are removed from higher ordinal pods to minimize resource usage [CORE]
+
+**Example ConfigMap for config-descheduler:**
+
+```
+data:
+  priorities: |+
+                  [
+                    {"Name": "RemoveWithEvenPodSpreadPriority",
+                    "Weight": 10,
+                    "Args": "{\"MaxSkew\": 2}"},
+                    {"Name": "RemoveWithAvailabilityZonePriority",
+                    "Weight": 10,
+                    "Args":  "{\"MaxSkew\": 2}"},
+                    {"Name": "RemoveWithHighestOrdinalPriority",
+                    "Weight": 2}
+                  ]
+```
+
+## Normal Operation
+
+1. **Busy scheduler**:
+
+Scheduler can be very busy allocating the best placements for multiple eventing sources at a time using the scheduler predicates and priorities configured. During this time, the cluster could see statefulset replicas increasing (as the autoscaler computes how many more pods are needed to complete scheduling) or replicas decreasing during idle time (as eviction kicks in to attempt compacting the vreplicas into a smaller number of pods or as KEDA lowers the number of vreplicas). The current placements are stored in the eventing source's status field for observability.
+
```

Comment:
Instead of: `replicas decreasing during idle time (as eviction kicks in to attempt compacting the vreplicas into a smaller number of pods or as KEDA lowers the number of vreplicas).`, what about something like that:

`replicas decreasing during idle time, either caused by less events flowing through the system, or the vreplicas compaction or the deletion of sources.`. 

I would avoid mentioning KEDA here (it's fine to mention it as an example).